<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMC GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTestUDP">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Message: </string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpMC">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>MC Data</string>
    </property>
    <widget class="QLabel" name="lblElapsedRuntime">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elapsed Runtime: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPitchActualPos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pitch Actual Position: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPitchActualVel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pitch Actual Velocity: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRollActualPos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Roll Actual Position: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRollActualVel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Roll Actual Velocity: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblYawActualPos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Yaw Actual Position: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblYawActualVel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Yaw Actual Velocity: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHorizontalActualPos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal Actual Position: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHorizontalActualVel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal Actual Velocity: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHeaveActualPos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Heave Actual Position: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHeaveActualVel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Heave Actual Velocity: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPlanetaryActualPos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Planetary Actual Position: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPlanetaryActualVel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Planetary Actual Velocity: </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpEMC">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>EMC Data</string>
    </property>
    <widget class="QLabel" name="lblPitchCommand">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pitch Comand:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRollCommand">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Roll Comand:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblYawCommand">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Yaw Command:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHorizontalCommand">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal Command:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHeaveCommand">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Heave Command:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPlanetaryCommand">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Planetary Command: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSetPoint">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VDC Set Point:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBlackoutStatus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VDS Blackout Status:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpSettings">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>EMC Settings Selections</string>
    </property>
    <widget class="QLabel" name="lblPitchSetting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pitch:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRollSetting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Roll:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblYawSetting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Yaw:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHorizSetting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHeaveSetting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Heave:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPlanetarySetting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Planetary:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnEMCCommand">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWaiting">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Waiting for MC setup...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirm">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
